indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorSpec.java

```diff
@@ -151,6 +157,37 @@ public DruidMonitorSchedulerConfig getMonitorSchedulerConfig()
   @Override
   public abstract Supervisor createSupervisor();
 
+  /**
+   * need to notice that autoScaler would be null which means autoscale is dissable.
+   * @param supervisor
+   * @return autoScaler, disable autoscale will return dummyAutoScaler and enable autoscale wiil return defaultAutoScaler by default.
+   */
+  @Override
+  @SuppressFBWarnings(value = "RV_RETURN_VALUE_IGNORED", justification = "using siwtch(String)")
+  public SupervisorTaskAutoscaler createAutoscaler(Supervisor supervisor)
+  {
+    String dataSource = getId();
+    SupervisorTaskAutoscaler autoScaler = new DummyAutoScaler(supervisor, dataSource);
+    AutoScalerConfig autoScalerConfig = ingestionSchema.getIOConfig().getAutoscalerConfig();
+
+    // kinesis'autoscalerConfig is always null for now, So that kinesis will hold a DummyAutoScaler.
+    // only SeekableStreamSupervisor is supported here.
+    if (autoScalerConfig != null
```

Comment:
if by mistake one submits `autoScalerConfig` for kinesis supervisor then seems like `DefaultAutoScaler ` is created for kinesis also ?

Reply:
I see `null` is being passed from `KinesisSupervisorIOConfig` so it will be null

Reply:
reviewed till here

Reply:
Thanks.